portfolio/src/main/webapp/learn.js

```diff
@@ -0,0 +1,54 @@
+// More API functions here:
+// https://github.com/googlecreativelab/teachablemachine-community/tree/master/libraries/image
+
+// the link to your model provided by Teachable Machine export panel
+
+const URL = "https://teachablemachine.withgoogle.com/models/7DRJsYZUY/";
+
+let model, webcam, labelContainer, maxPredictions;
+
+// Load the image model and setup the webcam
+async function init() {
+  const modelURL = URL + "model.json";
+  const metadataURL = URL + "metadata.json";
+
+  // load the model and metadata
+  // Refer to tmImage.loadFromFiles() in the API to support files from a file picker
+  // or files from your local hard drive
```

Comment:
was this copied from somewhere? if you're not using loadFromFiles() no need to reference it